src/binary/non_blocking/raw_binary_reader.rs

```diff
@@ -1283,7 +1283,7 @@ impl<'a, A: AsRef<[u8]>> TxReader<'a, A> {
 #[cfg(test)]
 mod tests {
     use crate::binary::non_blocking::raw_binary_reader::RawBinaryBufferReader;
-    use crate::text::text_value::IntoAnnotations;
+    use crate::text::text_value::IntoRawAnnotations;
```

Comment:
🗺️ The `IntoAnnotations` trait name was a bit misleading once I introduced an `Annotations` type. I have renamed it `IntoRawAnnotations`, reflecting the fact that it accepts raw symbol tokens.